src/components/v5/common/CompletedAction/partials/StreamingPayment/partials/StreamingPaymentWidget/StreamingPaymentWidget.tsx

```diff
@@ -97,6 +97,14 @@ const StreamingPaymentWidget: FC<StreamingPaymentWidgetProps> = ({
 
   const [hasEnoughFunds, setHasEnoughFunds] = useState(checkIfHasEnoughFunds());
 
+  useEffect(() => {
+    setHasEnoughFunds(checkIfHasEnoughFunds());
+  }, [
+    checkIfHasEnoughFunds,
+    amounts.amountAvailableToClaim,
+    streamingPaymentData,
+  ]);
```

Comment:
This dependency array looks like it only needs `checkIfHasEnoughFunds` in it, since that function comes from a useCallback inside a hook, which has the other variables in it's dependency array.

Also, a side note: personally I'm not a huge fan of the way that hook is set up, I feel like it could return a boolean of `hasEnoughFunds` rather than a callback. It makes the code just that bit harder to read and a little bit harder to debug and maintain. But anyway, it's unrelated to this PR so no worries, this is just the first time I saw it so it's the place where I'll make my comment 🤣